File: src/main/java/com/redgrapefruit/mythicaltowers/common/tool/weapon/DoubleAxeItem.java
package com.redgrapefruit.mythicaltowers.common.tool.weapon;

import com.redgrapefruit.mythicaltowers.common.core.EffectConfig;
import net.minecraft.item.ToolMaterial;

import java.util.List;

/**
 * A double axe takes longer to strike, but hits MUCH harder
 */
public class DoubleAxeItem extends CustomAxeItem {
    public DoubleAxeItem(List<EffectConfig> configs, ToolMaterial material, int attackDamage, float attackSpeed) {
        super(configs, material, attackDamage, attackSpeed);
    }
}

File: src/main/kotlin/com/redgrapefruit/mythicaltowers/tool/weapon/DoubleAxeItem.kt
package com.redgrapefruit.mythicaltowers.tool.weapon

import com.redgrapefruit.mythicaltowers.core.EffectConfig
import net.minecraft.item.ToolMaterial

/**
 * A double axe takes longer to strike, but hits MUCH harder
 */
class DoubleAxeItem(configs: List<EffectConfig>, material: ToolMaterial?, attackDamage: Int, attackSpeed: Float) :
    CustomAxeItem(configs, material, attackDamage, attackSpeed)